bookmarks.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/styles.css" />
+    <title>Quizz-App</title>
+  </head>
+  <body>
+    <header>
+      <h1>Quizz App</h1>
+    </header>
+    <main id="home">
+      <section class="question-box">
+        <img src="./assets/images/bookmark-dark.svg" class="bookmark" />
+        <h2 class="question">What property flips the axes in flexbox?</h2>
+        <button class="button">Show Answer</button>
+        <ul class="tags">
+          <li class="one-tag">#html</li>
```

Comment:
As a naming convention you could use "tag-item" instead. But with BEM it will be probably different as well.

Reply:
thx. changed it and will see for BEM later again

Reply:
You changed it in your CSS, now you have to change it everywhere it is used.